src/oneconfig/java/io/github/moulberry/notenoughupdates/compat/oneconfig/OneMoulConfig.java

```diff
@@ -0,0 +1,198 @@
+					optionField,
+					categoryInstance,
+					annotation.name(),
+					annotation.desc(),
+					cat, subcategory, 1
+				));
+			}
+			ConfigEditorText configEditorText = optionField.getAnnotation(ConfigEditorText.class);
+			if (configEditorText != null) {
+				category.options.add(new ConfigTextBox(
+					optionField,
+					categoryInstance,
+					annotation.name(),
+					annotation.desc(),
+					cat, subcategory, 1,
+					annotation.name(), false, false
+				));
+			}
+			ConfigEditorKeybind configEditorKeybind = optionField.getAnnotation(ConfigEditorKeybind.class);
+			if (configEditorKeybind != null) {
+				category.options.add(new OneFancyKeybind(
+					optionField,
+					categoryInstance,
+					annotation.name(),
+					annotation.desc(),
+					cat, subcategory, 1
+				));
+			}
+			ConfigEditorColour configEditorColour = optionField.getAnnotation(ConfigEditorColour.class);
+			if (configEditorColour != null) {
+				category.options.add(new OneFancyColor(
+					optionField,
+					categoryInstance,
+					annotation.name(),
+					annotation.desc(),
+					cat, subcategory, 1, true
+				));
+			}
+			ConfigEditorSlider configEditorSlider = optionField.getAnnotation(ConfigEditorSlider.class);
+			if (configEditorSlider != null) {
+				category.options.add(new WrappedConfigSlider(
+					optionField,
+					categoryInstance,
+					annotation.name(),
+					annotation.desc(),
+					cat,
+					subcategory,
+					configEditorSlider.minValue(),
+					configEditorSlider.maxValue(),
+					(int) configEditorSlider.minStep()
+				));
+			}
+			ConfigEditorButton configEditorButton = optionField.getAnnotation(ConfigEditorButton.class);
+			if (configEditorButton != null) {
+				category.options.add(new ConfigButton(
+					() -> moulConfig.executeRunnable(configEditorButton.runnableId()),
+					categoryInstance,
+					annotation.name(),
+					annotation.desc(),
+					cat,
+					subcategory,
+					1, configEditorButton.buttonText()
+				));
+			}
+			ConfigEditorDropdown configEditorDropdown = optionField.getAnnotation(ConfigEditorDropdown.class);
+			if (configEditorDropdown != null) {
+				category.options.add(new ConfigDropdown(
+					optionField,
+					categoryInstance,
+					annotation.name(),
+					annotation.desc(),
+					cat, subcategory,
+					1, configEditorDropdown.values()
+				));
+			}
+			ConfigEditorDraggableList configEditorDraggableList = optionField.getAnnotation(ConfigEditorDraggableList.class);
+			if (configEditorDraggableList != null) {
+				category.options.add(new ConfigInfo(
+					optionField, categoryInstance,
+					"This option (" + annotation.name() + ") is not available via the oneconfig gui",
+					cat, subcategory, 1, InfoType.ERROR
+				));
+			}
+			ConfigEditorFSR configEditorFSR = optionField.getAnnotation(ConfigEditorFSR.class);
+			if (configEditorFSR != null) {
+				category.options.add(new ConfigInfo(
+					optionField, categoryInstance,
+					annotation.desc(), cat, subcategory, 1, InfoType.WARNING
```

Comment:
make info size 2 (and do the same to some other options iyw) to avoid text cutting out